Lib/inspect.py

```diff
@@ -173,6 +177,13 @@
 TPFLAGS_IS_ABSTRACT = 1 << 20
 
 
+@enum.global_enum
+class AnnotationsFormat(enum.IntEnum):
```

Comment:
How does this PR relate to PEP 749? Are you intending for this PR to implement PEP 649 as written, or the updated proposals in PEP 749?

Reply:
I will just take out this part; I only use the enum values in one test. The companion PR #119891 will implement PEP 749's proposed new module.

Reply:
And thanks for reviewing!